examples/forecasting_strategies.ipynb

```diff
@@ -0,0 +1,954 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Forecasting stratigies\n",
+    "\n",
+    "This notebook describes some strategies of foreacsting.\n",
+    "\n",
+    "Notebook navigation:\n",
+    "- [Intro](#20-what-are-strategies)\n",
+    "- [Pipeline](#21-pipeline)\n",
+    "- [AutoregressivePipeline](#22-autoregressivepipeline)\n",
+    "- [DirectEnsemble](#23-directensemble)\n",
+    "- [assemble_pipelines + DirectEnsemble](#24-assemble-pipelines-with-directensemble)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Load Dataset\n",
+    "\n",
+    "In this notebook we will work with the dataset contains only one segment with monthly wine sales. Working process with the dataset containing more segments will be absolutely the same."
```

Comment:
Let's work here with example_dataset.csv, it is already in the right format and the load process will rake less place